resources/js/components/ObservationList.vue

```diff
@@ -231,6 +331,9 @@ export default {
 }
 </script>
 
-<style scoped>
+<style>
 
+.background-color-on-selection.form-control-multiselect .multiselect .multiselect__tags {
+  background:lightgrey;
+}
```

Comment:
Ich bin nicht zu 100% sicher, dass die Komponenten-Styles in unserem Deployment funktionieren. Um sicherzugehen könntest du das hier auch stattdessen in app.scss verschieben.